dffml/accuracy/accuracy.py

```diff
@@ -24,9 +22,7 @@ def __init__(self, parent: "Accuracy") -> None:
         self.parent = parent
 
     @abc.abstractmethod
-    async def score(
-        self, mctx: ModelContext, sources: SourcesContext,
-    ) -> float:
+    async def score(self, mctx, sources) -> float:
```

Comment:
I think we can remove this commit

```console
$ git rebase -i HEAD~8
```

Then drop the commit.

Should be able to replace it with removal of `from ..accuracy.accuracy import AccuracyContext` line from `dffml/model/model.py`.